docs/admin/database.rst

```diff
@@ -23,13 +23,198 @@ Migrating MySQL Data to PostgreSQL
     This results in a database schema that is not compatible with future migrations.
 
     PgLoader converts constraint names into all-lowercase.
-    The Liquibase migrations assume that they have got their original name which contains the prefix ``FK``.
+    The Liquibase migrations assume that they have got their original name which contains the case-sensitive prefix ``FK``.
 
 
-.. note::
-    Start Artemis at least once in version 5.12.9 or greater to make sure the current database schema is PostgreSQL-compatible.
-    Only Artemis 6.0.0 or newer can connect to a PostgreSQL database.
+#. Start Artemis at least once in version ``V`` ≧ 6.0.0 or greater to make sure the current database schema is PostgreSQL-compatible.
 
+#. Stop Artemis.
+
+#. Create a database backup using ``mysqldump``.
+   This dump is called ``Artemis.sql`` in the following steps.
+
+#. Copy the ``docker-compose.yml`` file into the same directory as the ``Artemis.sql`` database dump
+   and run the following commands to convert the ``Artemis.sql`` dump into ``Artemis.pg.sql`` that is usable by PostgreSQL.
+
+   .. code-block:: yaml
+        :caption: ``docker-compose.yml`` with helper containers for MySQL and PostgreSQL.
+
+        ---
+        services:
+            mysql:
+                image: docker.io/library/mysql:8
+                environment:
+                    - MYSQL_ROOT_PASSWORD=cRS2N4X2
```

Comment:
This password is only used locally for the database containers that are created for this task and destroyed afterwards. So there is just some randomly generated password since PostgreSQL doesn’t allow an empty one.
The password is completely independent of the one for the user’s actual system.